### PR TITLE
New active docs tweak

### PR DESCRIPTION
**Pull request by josemigallas**
2 minor changes to the "new active doc" page.
Current:
Screenshot 2020-12-11 at 18 49 05

New:
Screenshot 2020-12-11 at 18 46 50

### CODE REVIEW

File: app/views/admin/api_docs/base/_form.html.slim [hunk 1]
```diff
@@ -15,7 +15,7 @@
   = form.semantic_errors :base_path
   = form.input :skip_swagger_validations, as: :boolean
 = form.buttons do
-  = form.commit_button
+  = form.commit_button 'Create Spec'
```
Comment (damianpm) [inline app/views/admin/api_docs/base/_form.html.slim:18]: If not from here, where is coming from the "Create Service" text in the button?

Comment (josemigallas) [reply app/views/admin/api_docs/base/_form.html.slim:18]: It's inferred from the model, the form is for `ApiDocs::Service`: https://github.com/3scale/porta/blob/9f413e8361f2752c47c44b4b9beaab27f5c0669a/app/views/admin/api_docs/base/new.html.erb#L3

**Review by macejmic — APPROVED** [review]

**Review by damianpm — APPROVED** [review]

**Review by damianpm — CHANGES_REQUESTED** [review]
Test failing

File: app/controllers/admin/api_docs/base_controller.rb [hunk 2]
```diff
@@ -12,6 +12,7 @@ def index
 
   def new
     @api_docs_service = api_docs_services.new
+    @commit_button_label = 'Create Spec'
```
Comment (Martouta) [inline app/controllers/admin/api_docs/base_controller.rb:15]: Do we need an instance variable in the controller for this? This either goes to the view directly or at least a helper method that returns 'Create Spec' or 'Update Spec' depending on the `action_name` for the view to use

**Review by Martouta — CHANGES_REQUESTED** [review]
Unnecessary instance variable in the controller for a string in the view 🙈

File: app/views/admin/api_docs/base/_form.html.slim [hunk 3]
```diff
@@ -1,6 +1,8 @@
 - content_for :javascripts do
   = javascript_include_tag 'provider/codemirror'
 
+- commit_button_label = action_name == 'edit' ? 'Update Spec' : 'Create Spec'
```
Comment (Martouta) [inline app/views/admin/api_docs/base/_form.html.slim:4]: Safer:
```suggestion
- commit_button_label = @api_docs_service.new_record? ? 'Create Spec' : 'Update Spec'
```

**Review by Martouta — DISMISSED** [review]

**Review by damianpm — DISMISSED** [review]

**Review by Martouta — APPROVED** [review]

**Pull request merged**